### ISSUE TITLE
Track Thymeleaf 3.0

### ISSUE THREAD

**Issue by opensource21** [theborakompanioni/thymeleaf-extras-shiro#13]
Thymeleaf 3.0 is on the way to become final. Unfortunately it's some work for dialects
see http://www.thymeleaf.org/doc/articles/thymeleaf3migration.html

**Comment by theborakompanioni** [comment]
Uff. This is more work than I expected it to be -> very poor documentation for v3 at the moment. I'll keep you updated as soon as progress is made.

**Comment by opensource21** [comment]
Perhaps it's better to wait, till the situation at documentation become better. Have you read http://forum.thymeleaf.org/Thymeleaf-3-dialect-migration-guide-td4029602.html ?

**Comment by theborakompanioni** [comment]
@opensource21 definitely - let's see when things improve. Like I said - I'll keep you updated. Thank you for the reminder!

**Comment by saviola777** [comment]
Any updates? I'm currently using raw calls like

```HTML
<th:block th:if="${T(at.pollux.thymeleaf.shiro.dialect.ShiroFacade).isGuest()}">
```

because the tags aren't working with 3.0.

**Comment by theborakompanioni** [comment]
@saviola777 Sorry. No updates yet. Documentation is still poor. Any chance someone is willing to work on it? Pull requests are very welcome!

**Comment by opensource21** [comment]
Perhaps the easiest way is to copy the code from [spring-security](https://github.com/thymeleaf/thymeleaf-extras-springsecurity/tree/3.0-dev/thymeleaf-extras-springsecurity4/src) and adopt it to shiro?

**Comment by rline1** [comment]
i have

**Comment by logemann** [comment]
whats the status on that? Could perhaps spent some time to help....

**Comment by theborakompanioni** [comment]
@logemann Thank you! Much appreciated! There is an open pull request (#14) from @rline1. I have not had the chance to fully review it but it would be awesome if you'd take a look at it!

**Comment by theborakompanioni** [comment]
Merged #14 and released as snapshot of 2.0.0

**Comment by theborakompanioni** [comment]
2.0.0 has been released.
